UnitsNet.Tests/CustomCode/VolumeTests.cs

```diff
@@ -48,14 +48,24 @@ public class VolumeTests : VolumeTestsBase
         protected override double CubicYardsInOneCubicMeter => 1.30795062;
 
         protected override double DecilitersInOneCubicMeter => 1E4;
+        protected override double HectocubicFeetInOneCubicMeter => 3.531472e-1;
+        protected override double HectocubicMetersInOneCubicMeter => 0.01;
 
         protected override double HectolitersInOneCubicMeter => 1E1;
 
         protected override double ImperialGallonsInOneCubicMeter => 219.96924;
 
         protected override double ImperialOuncesInOneCubicMeter => 35195.07972;
 
+        protected override double KilocubicFeetInOneCubicMeter => 3.531472e-2;
```

Comment:
Wait, is this really a unit? Kilocubic feet? Sure this is not just 1000 cubicfeet? I can't find kilocubic feet on google.

Reply:
Hi @angularsen This is a unit of measure, that used in this product https://portfoliomanager.energystar.gov/ and called **kcf (thousand cubic feet)**
And, yeah, it sounds weird for me too, but guys in the States seems to be absolutely in love with some strange units of measure. Take it or leave it - it's up to you of course.